dbt/dbt_project.yml

```diff
@@ -27,44 +27,17 @@ clean-targets:
 models:
   athena:
     +materialized: view
-    ccao:
-      # Set default materialization on a per-schema basis. Schemas that contain
-      # _any_ persistent models should be set to `view` or `table`, or left
-      # to inherit the `view` default which is set on the `athena` project, but
-      # schemas that are entirely ephemeral should be marked `ephemeral`. The
-      # point of this pattern of defaults is that schemas with non-ephemeral
-      # defaults will be marked for deletion by the cleanup_dbt_resources.sh
-      # script, since `dbt list --select config.materialization:view` only
-      # reads this config file and does not consider model-level overrides.
-      # Hence, only schemas that contain exclusively ephemeral models should
-      # be marked with an `ephemeral` default, since we won't build those
-      # schemas into Athena and we don't want cleanup_dbt_resources.sh to
-      # try to clean them up.
-      +materialized: ephemeral
-      +schema: ccao
```

Comment:
This config is no longer required, since we don't need to configure materialization or schemas for sources (schemas are inferred from the source name thanks to the dbt-athena plugin). If and when we materialize these sources, we can add these blocks back in, but for now I removed them to suppress warnings about unused configs in the dbt CLI.

Reply:
**praise:** So this change actually _reduces_ the complexity of the previous implementation. Awesome find!